src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const listBlock = document.querySelector('.list-block');
+const task = [
+  {
+    description: 'Wash the dishes',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Clean the house',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Cut the grass',
+    completed: false,
+    index: 3,
+  },
+];
+
```

Comment:
- [ ] **Required:** Great job implementing this. 👏 However, the To-do list should be rendered in the order of their index. Kindly sort by their indexes before displaying.
Consider this:
`Array.sort((a, b) => a.index - b.index);`